.github/workflows/build.yml

```
@@ -12,7 +12,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: gradle/wrapper-validation-action@v1
-      - run: ./gradlew build javadocJar checkMappings --stacktrace
+      - run: |
+          curl https://maven.fabricmc.net/net/java/jdk/openjdk-17-ea%2B24_linux-x64_bin.tar.gz --output java.tar.gz
+          tar -xf java.tar.gz
+      - run: ./gradlew build javadocJar checkMappings --stacktrace -Porg.gradle.java.installations.paths=./jdk-17
```

Maybe we can debug by like:
```suggestion
      - run: ./gradlew javaToolchains build javadocJar checkMappings --stacktrace -Porg.gradle.java.installations.auto-download=false -Porg.gradle.java.installations.paths=./jdk-17
```